### DISCUSSION TITLE
Tauri app with sqlite

### DISCUSSION THREAD

**Discussion by RodrigoMindera** [tauri-apps/tauri#10612] [Q&A]
I created an app using Tauri and the sql plugin, and everything works like a charm. But I was surprised when I created an installer , that all data I inserted to test my app was there. How can I avoid that?

**Comment by FabianLars** [comment]
This is only the case on your system. If you share the installer with others they will have a fresh database. Currently the plugin forces a path in the appData folder no matter if you're running tauri dev or the built app. At some point it'll support custom paths so you could change it yourself then.
